pkg/install/generate.go

```diff
@@ -700,8 +722,18 @@ func completeVSphereDeprovisionJob(req *hivev1.ClusterDeprovision, job *batchv1.
 			ImagePullPolicy: images.GetHiveImagePullPolicy(),
 			Env:             env,
 			Command:         []string{"/bin/sh", "-c"},
-			Args:            []string{fmt.Sprintf("cp -vr %s/. /etc/pki/ca-trust/source/anchors/ && update-ca-trust && /usr/bin/hiveutil deprovision vsphere --vsphere-vcenter %s --loglevel debug %s", vsphereCloudsDir, req.Spec.Platform.VSphere.VCenter, req.Spec.InfraID)},
-			VolumeMounts:    volumeMounts,
+			Args: []string{
+				fmt.Sprintf(
+					"cp -vr %s/. /etc/pki/ca-trust/source/anchors/ && "+
+						"update-ca-trust && "+
+						"/usr/bin/hiveutil deprovision vsphere --vsphere-vcenter %s --loglevel debug --creds-dir=%s %s",
+					vsphereCloudsDir,
+					req.Spec.Platform.VSphere.VCenter,
+					"/vsphere-creds",
```

Comment:
can we constify this like we've done for the other platforms?

Reply:
We could, but it is local to this function. I can create a function-scoped const.